packages/lesswrong/components/languageModels/LanguageModelLauncherButton.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback, useEffect } from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useTracking } from "../../lib/analyticsEvents";
+import { useDialog } from '../common/withDialog';
+import { useCookiesWithConsent } from '../hooks/useCookiesWithConsent';
+import { SHOW_LLM_CHAT_COOKIE } from '@/lib/cookies/cookies';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    position: "fixed",
+    bottom: 20,
+    right: 80,
+    zIndex: theme.zIndexes.languageModelChatButton,
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "center",
+    borderRadius: "50%",
+    width: 48,
+    height: 48,
+    padding: 10,
+    cursor: "pointer",
+    // TODO: Figure out for dark mode
+    // backgroundColor: theme.palette.grey[100],
```

Comment:
^

Reply:
I've edited the comment. The remaining stuff that doesn't work on darkmode is the shadow and hover effect